3.10/aql/graphs-all-shortest-paths.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: default
+description: This type of query finds all shortest paths between two given documents(startVertex and targetVertex) in your graph
+---
+All Shortest Paths in AQL
+=======================
+
+General query idea
+------------------
+
+This type of query finds *all* shortest paths between two given documents, *startVertex* and *targetVertex* in your graph.
+
+Every such path will be returned as a JSON object with two components:
+
+- an array containing the `vertices` on the path
+- an array containing the `edges` on the path
+
+Currently, there is no support yet for a `weightAttribute` in All Shortest Paths `OPTIONS`.
+
+<!-- {% include youtube.html id="XdITulJFdVo" %} -->
+
+**Example**
+
+Let us take a look at a simple example to explain how it works.
+We will using the following social networkx graph to demonstrate all shortest paths:
+This is the graph that we are going to find all shortest paths on:
+
+![Social Network](../images/social_graph_2.png)
```

Comment:
note: the following social network graph is used as a placeholder for a real `ALL_SHORTEST_PATHS` use case example